.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "env": {
+    "node": true
+  },
+  "rules": {
+    "indent": [2, 4, {"SwitchCase": 1}],
```

Comment:
Having had a quick look as eslint - I would also suggest:

`````
one-var: [2, "never"]
`````

As that was always our preferred style (I've added it to the contributing page on the Wiki as it was missing).